static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -391,3 +395,52 @@ export function onPointClicked(
     .style("top", top + "px")
     .style("visibility", "visible");
 }
+
+/**
+ * Sets the url hash with the given key and value
+ * @param paramKey the param key to update in the hash
+ * @param paramValue the value to use in the hash
+ */
+export function setUrlHash(paramKey: string, paramValue: string): void {
+  // TODO (chejennifer): Right now we are assuming each subject page only has
+  // one disaster event map tile. Find a way to handle the url hash for
+  // mulitple disaster event map tiles.
+  const urlParams = new URLSearchParams(window.location.hash.split("#")[1]);
+  urlParams.set(paramKey, paramValue);
+  window.location.hash = urlParams.toString();
+}
+
+/**
+ * Gets the date for a disaster event map using url params.
+ */
+export function getDate(): string {
+  const urlParams = new URLSearchParams(window.location.hash.split("#")[1]);
+  return urlParams.get(URL_HASH_PARAM_KEYS.DATE) || DATE_OPTION_6M_KEY;
+}
+
+/**
+ * Gets the severity filters for a disaster event map using url params.
+ * @param eventTypeSpec the event type spec for the disaster event map
+ */
+export function getSeverityFilters(
+  eventTypeSpec: Record<string, EventTypeSpec>
+): Record<string, SeverityFilter> {
+  const urlParams = new URLSearchParams(window.location.hash.split("#")[1]);
+  let urlSeverityFilters = {};
+  if (urlParams.get(URL_HASH_PARAM_KEYS.SEVERITY_FILTER)) {
```

Comment:
nit: add the return value  to a local var, then test that var? instead of calling `.get` twice.